TrafficSimulator/Assets/Car/AutoDrive.cs

```diff
@@ -69,11 +77,19 @@ public class AutoDrive : MonoBehaviour
         private LaneNode _startNode;
         private LaneNode _endNode;
         private LaneNode _currentNode;
+        private Vector3 _prevIntersectionPosition;
+        private NavigationNode _navigationPathEndNode;
+        private Stack<NavigationNodeEdge> _navigationPath = new Stack<NavigationNodeEdge>();
 
         public LaneNode CustomStartNode = null;
 
+        private GameObject _navigationPathContainer;
+        
         void Start()
         {
+            Road.RoadSystem.Setup();
+            _navigationPathContainer = new GameObject("Navigation Path");
```

Comment:
Dunno

Reply:
This will have to be redone later. If it is set as a container on the car then the marker will follow the car as the position is relative to the container, and the container moves with the car if it is included in the Car prefab as of now